### ISSUE TITLE
You successfully completed the dart to LLVM build project

### ISSUE THREAD

**Issue by insinfo** [linzj/llvm-toy#14]
You successfully completed the dart to LLVM build project ?
https://groups.google.com/a/dartlang.org/g/misc/c/9I2hypnR6BE

**Comment by linzj** [comment]
Yes. Internally in alibaba group we have used llvm aot build for 2 years.

**Comment by insinfo** [comment]
@linzj
Could this be published in a public repository for others to use? Or is it just a private project?

**Comment by linzj** [comment]
> @linzj Could this be published in a public repository for others to use? Or is it just a private project?


We have currently only opened the code to certain members of the Dart VM team.

**Comment by insinfo** [comment]
@linzj
I see, hopefully at some point this will be available as open source, thanks for reaching out to me and I hope your project moves forward, good luck
